.github/workflows/qa.yaml

```diff
@@ -150,59 +128,3 @@ jobs:
         if: startsWith(github.ref, 'refs/tags/')
         with:
           files: installer/Output/adwatchd_setup.exe
-
-  generators:
-    name: Generated files are up to date
```

Comment:
Are we dropping this entirely?

Reply:
Look at the reused action. It has the generator :)

Reply:
Oh, nice!